css-text-4/Overview.bs

```diff
@@ -4603,8 +4456,41 @@ Breaking Rules for Letters: the 'word-break' property</h3>
 			(which uses [=spaces=] between words),
 			and is also useful for mixed-script text where CJK snippets are mixed
 			into another language that uses [=spaces=] for separation.
+
+		<dt><dfn>phrase</dfn>
+		<dd>
+			This value directs the user agent to perform language-specific content analysis
+			to prioritize keeping natural phrases (of multiple words) together
+			to determine [=soft wrap opportunities=]
+			and [=virtual word boundaries=].
+
+            User agents may activate
+            language-specific content analysis
+            in response to user preferences.
+            User agents with this behavior must do this
+            by setting the [=declared value=] of 'word-break' to ''word-break/phrase''
+            in the [=User Origin=].
+            User agents that do not support the [=User Origin=]
+            should use the [=User-Agent Origin=] instead.
+
+            Note: This means that web content can detect whether or not this feature is
+            enabled by calling getComputedStyle(), even if the user agent enabled this
+            feature by default.
 	</dl>
 
+	<div class=advisement>
+		Manual analysis of the content can be more reliable than UA heuristics.
+		For best results, authors who can perform this analysis are encouraged to markup their documents
+		using <{wbr}> or U+200B
+		to exhaustively indicate word boundaries.
```

Comment:
This should probably add that UAs should respect such WBRs as more authoritative than their automatic detection.